deploy/helm/kuberhealthy/templates/khcheck-dns.yaml

```diff
@@ -9,6 +9,9 @@ spec:
   runInterval: 2m
   timeout: 15m
   podSpec:
+    securityContext:
```

Comment:
Okay to add a if enabled = true for each check?
This way I can disable the security context and let OCP handle it for me.
Else I will have to see the error unable to create with user id 999 okay range is something and then I would have to go in and fix it in helm.

Easy enough to do but It will be a pain if I want to do gitops in the future ^^

Reply:
The important part is to be able to enable/disable
    securityContext:
      runAsUser: {{ .Values.securityContext.runAsUser }}
      fsGroup: {{ .Values.securityContext.fsGroup }}

I don't have any need to disable 
        securityContext:
          allowPrivilegeEscalation: {{ .Values.securityContext.allowPrivilegeEscalation }}
          readOnlyRootFilesystem: {{ .Values.securityContext.readOnlyRootFilesystem }}
But i guess consistency is nice, or connect both of them to the same check.

Reply:
Can we also add the same if statement to the kuberhealthy deploy file in helm while we are at it :)

Reply:
Cool I can go ahead and add that in

Reply:
Thanks these changes looks good.
But can you add the same if statement to deployment.yaml?